docs/contributing.md

```diff
@@ -0,0 +1,136 @@
+# Contributing
+
+This document describes how to install KubeSaw in a development environment.
+
+## Prerequisites
+
+### OpenShift Cluster
+Ensure you have access to an OpenShift 4.6+ cluster with cluster admin privileges and log in using `oc login`
+
+### Required Tools
+Install the [required tools](required-tools.md).
+
+### Authentication
+Configure authentication for the cluster using one of the following options:
+
+1. Contact a member of the KubeSaw Team for instructions on how to configure the cluster to use our internal Dev SSO.
```

Comment:
Given that we're aiming for Kubesaw to be installable by the 3rd parties, I think this should not be mentioned as the 1st option. We could leave it in as the last option and prepend it with something like "If you're a Redhatter, you can also ...".

Reply:
Good point! I'll do that. Even if TBH this is for now just an INTERNAL Contribution guide. I would expect that we have a complete new/different one once we open for external contributions.

Reply:
done in: https://github.com/kubesaw/kubesaw.github.io/pull/10/commits/f7fc24c415cfc0f65cf0a928707b8fc569578319